cmd/set.go

```diff
@@ -52,3 +74,8 @@ func SetCommand() cli.Command {
 		},
 	}
 }
+
+func SetProxyPort(port int) (p string) {
+	p = fmt.Sprintf("http://localhost:%v", port)
+	return
```

Comment:
this would be better done without the named result parameter and just:
```
    return fmt.Sprintf("http://localhost:%d", port)
```

Note the change of `%v` to `%d`. Port is an `int`, so the more tightly-constrained `%d` makes more sense.

Reply:
Done